### PR TITLE
completed css-flexbox

### PR DESCRIPTION
**Pull request by josephswpark**
https://user-images.githubusercontent.com/113474457/195468377-ddb12424-1560-4d6e-bee1-07590a9ec5bf.mp4

### CODE REVIEW

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Hey Joseph, your output in the gif does not match the output examples in the exercise, please revise.

**Comment by josephswpark** [comment]
Screen Shot 2022-10-12 at 8 00 24 PM

File: css-flexbox/align-items.html [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Flexbox - align-items</title>
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="pokemon.css">
+  <style>
+    /* your styles here 👇 */
+    #card-deck {
+      display: flex;
+      flex-basis: calc(100%/ 3);
```
Comment (codingcodymiller) [inline css-flexbox/align-items.html]: This should be on `card-wrapper`

File: css-flexbox/justify-content.html [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Flexbox - justify-content</title>
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="pokemon.css">
+  <style>
+    /* your styles here 👇 */
+    #card-deck {
+      display: flex;
+      flex-basis: 25%;
```
Comment (codingcodymiller) [inline css-flexbox/justify-content.html:14]: This should be on `card-wrapper`

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**